src/sdk/pynni/nni/compression/torch/quantization/quantizers.py

```diff
@@ -110,12 +112,111 @@ def get_bits_length(config, quant_type):
     else:
         return config["quant_bits"].get(quant_type)
 
+def _check_weight(module):
+    try:
+        return isinstance(module.weight.data, torch.Tensor)
+    except AttributeError:
+        return False
+
+class Post_Training_Quantize():
```

Comment:
the coding style should be consistent with existing code. For example, `Quantize` -> `Quantizer`, `Post_Training_Quantize` -> `PostTrainingQuantize`

Reply:
it is also strange that it does not inherit the class `Quantizer`

Reply:
I think the implementation of class `Quantizer` is designed for 'quantize aware training'. It is not suitable for post-training quantization and mixed bit-width quantization.